tests/test_mediastore/test_mediastore.py

```diff
@@ -192,3 +193,52 @@ def test_get_metric_policy_raises_error_if_container_does_not_have_metric_policy
     with pytest.raises(ClientError) as ex:
         client.get_metric_policy(ContainerName="container-name")
     ex.value.response["Error"]["Code"].should.equal("PolicyNotFoundException")
+
+
+@mock_mediastore
+def test_list_tags_for_resource():
+    client = boto3.client("mediastore", region_name=region)
+    tags = [{"Key": "customer"}]
+
+    create_response = client.create_container(
+        ContainerName="Awesome container!", Tags=tags
+    )
+    container = create_response["Container"]
+    response = client.list_tags_for_resource(Resource=container["Name"])
+    response["ResponseMetadata"]["HTTPStatusCode"].should.equal(200)
+    response["Tags"].should.equal(tags)
+
+
+@mock_mediastore
+def test_list_tags_for_resource_return_none_if_no_tags():
+    client = boto3.client("mediastore", region_name=region)
+
+    create_response = client.create_container(
+        ContainerName="Awesome container!"
+    )
+    container = create_response["Container"]
+    response = client.list_tags_for_resource(Resource=container["Name"])
+    response["ResponseMetadata"]["HTTPStatusCode"].should.equal(200)
+    response.get("Tags").should.equal(None)
+
+
+@mock_mediastore
+def test_delete_container():
+    client = boto3.client("mediastore", region_name=region)
+
+    create_response = client.create_container(
+        ContainerName="Awesome container!"
+    )
+    container = create_response["Container"]
+    response = client.delete_container(ContainerName=container["Name"])
+    response["ResponseMetadata"]["HTTPStatusCode"].should.equal(200)
+
```

Comment:
Can we verify that the deletion was actually successful? Easiest way is probably to verify that the container name does no longer show up in a `list_containers` call.

Reply:
Sure, in fact, I forgot to delete it from the containers list. Hope it's sorted now. Thanks so much!